.github/workflows/build-boost-packages.yml

```diff
@@ -10,19 +10,26 @@ on:
         description: 'Whether to publish releases'
         required: true
         default: 'false'
+  pull_request:
+    paths-ignore:
+    - 'versions-manifest.json'
+    - 'LICENSE'
+    - '**.md'
+    branches:
+    - 'main'
 
 env:
-  VERSION: ${{ github.event.inputs.VERSION }}
+  VERSION: ${{ github.event.inputs.VERSION || '1.73.0' }}
 defaults:
   run:
     shell: pwsh
 
 jobs:
   build_boost:
-    name: Build Boost ${{ github.event.inputs.VERSION }} [${{ matrix.os }}]
+    name: Build Boost ${{ github.event.inputs.VERSION || '1.73.0' }} [${{ matrix.os }}]
```

Comment:
Just for curiosity why is the 1.73.0 boost version?

Reply:
`1.73.0` is the latest Boost version. Do you have any concerns about this?

Reply:
Nope. 1.74.0 is the latest boost version - https://www.boost.org/users/history/version_1_74_0.html

Reply:
I meant `1.73.0` is the latest Boost version available from [versions-manifest.json](https://github.com/actions/boost-versions/blob/main/versions-manifest.json#L3).